### PR TITLE
Implement grid with glider visualization

### PR DESCRIPTION
**Pull request by LucasUTNFRD**
- Uses Macroquad for rendering
- Creates a fixed grid with a glider pattern

### CODE REVIEW

**Review by damiramirez — APPROVED** [review]
LGTM

File: src/conways.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+impl Grid {
+    pub fn new(width: usize, height: usize) -> Self {
+        Self {
+            grid: vec![vec![CellState::Dead; width]; height],
+        }
+    }
+
+    pub fn set(&mut self, x: usize, y: usize) {
+        self.grid[y][x] = CellState::Alive;
+    }
```
Comment (dsocolobsky) [inline src/conways.rs:20]: I would name this `set_alive` , since it's not clear what it's setting to

File: src/conways.rs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    pub fn get(&self, x: usize, y: usize) -> CellState {
+        self.grid[y][x].clone()
+    }
```
Comment (dsocolobsky) [inline src/conways.rs:24]: If the method doesn't do anything else why not do `grid.grid[y][x]` directly?

**Review by dsocolobsky — APPROVED** [review]
LGTM I left some minor comments for the next PR

**Pull request merged**